Labs/Lab_6/src/main/java/com/fmi/web/lab/raceeventmanagement/controller/EventController.java

```diff
@@ -0,0 +1,128 @@
+package com.fmi.web.lab.raceeventmanagement.controller;
+
+import com.fmi.web.lab.raceeventmanagement.model.Racer;
+import com.fmi.web.lab.raceeventmanagement.model.Team;
+import com.fmi.web.lab.raceeventmanagement.model.Track;
+import com.fmi.web.lab.raceeventmanagement.repository.sequence.TeamSequence;
+import com.fmi.web.lab.raceeventmanagement.service.EventService;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@RestController
+@RequestMapping("/controller")
+public class EventController {
+    @Autowired
+    EventService eventService;
+
+    @PostConstruct
+    public void initEventService() {
+        // Create a test track
+        Track testTrack = new Track("Speedway", 5000);
+        testTrack.setId(1);
+
+        // Create test teams and racers
+        Racer racer1 = new Racer("John", "Doe", 28);
+        Racer racer2 = new Racer("Jane", "Doe", 25);
+        List<Racer> team1Racers = Arrays.asList(racer1, racer2);
+        team1Racers.stream().forEach(team -> team.setId(TeamSequence.getNextValue()));
+
+        Team team1 = new Team("Team Lightning", Arrays.asList(racer1, racer2));
+
+        Racer racer3 = new Racer("Mike", "Smith", 30);
+        Racer racer4 = new Racer("Sarah", "Jones", 27);
+        List<Racer> team2Racers = Arrays.asList(racer3, racer4);
+        team2Racers.stream().forEach(team -> team.setId(TeamSequence.getNextValue()));
+
+        Team team2 = new Team("Team Thunder", Arrays.asList(racer3, racer4));
+
+        // Prepare a list of teams for the event
+        List<Team> testTeams = Arrays.asList(team1, team2);
+
+        // Specify the date and time for the event
+        LocalDateTime dateOfEvent = LocalDateTime.of(2024, 3, 21, 15, 0); // 21st March 2024, 3:00 PM
+
+        // Create event
+        eventService.createEvent("Spring Grand Prix", testTrack, testTeams, dateOfEvent);
+
+        // Print all events
```

Comment:
When working on a specific task, remove the unnecessary comment to keep your code clean